main.py

```diff
@@ -10,18 +10,80 @@
 
 log = logging.getLogger("mkdocs")
 
-FIND_REPLACE = {  # original relative to /docs : redirect target
-    "<CONTRIBUTING.md>": "[Contributing Section](development/)",
-    "(CODE_OF_CONDUCT.md)": "(development/#tides-code-of-conduct)",
-    "(CLA.md)": "(development/#tides-contributor-license-agreement)",
-    "CONTRIBUTING.md)": "development/)",
-    "<LICENSE>": "[LICENSE](https://github.com/TIDES-transit/TIDES/blob/main/LICENSE)",
-    "contributors.md)": "development/#contributors)",
-    "architecture.md)": "architecture)",
-    "tables.md": "tables",
-    "https://tides-transit.github.io/TIDES/governance/":"governance.md"
+BRANCH_NAME_KEYWORD = "{branch_name}"
+GITHUB_REPO = f"http://github.com/TIDES-transit/TIDES/tree/{BRANCH_NAME_KEYWORD}"
```

Comment:
Why do we need `GITHUB_REPO`? Shouldn't `./LICENSE` work instead of `https://github.com/TIDES-transit/TIDES/tree/main/LICENSE`? Or does that not work?